synapse/config/oidc.py

```diff
@@ -208,17 +208,22 @@ def generate_config_section(self, **kwargs: Any) -> str:
         #
         #   It is possible to configure Synapse to only allow logins if certain attributes
         #   match particular values in the OIDC userinfo. The requirements can be listed under
-        #   `attribute_requirements` as shown below. All of the listed attributes must
-        #   match for the login to be permitted. Additional attributes can be added to
-        #   userinfo by expanding the `scopes` section of the OIDC config to retrieve
-        #   additional information from the OIDC provider.
+        #   `attribute_requirements` as shown below. All of the listed in `value` attributes and
+        #   any of the listed in `values` attributes must match for the login to be permitted.
+        #   Additional attributes can be added to userinfo by expanding the `scopes` section
+        #   of the OIDC config to retrieve additional information from the OIDC provider.
+        #
+        #   If you specify both `value` and `values` entries, then only `value` entry will be checked
+        #   and `values` will be ignored.
```

Comment:
```suggestion
        #   If you specify both `value` and `values` entries, then only the `value` entry will be checked
        #   and `values` will be ignored.
```

Reply:
since you're using `oneOf` in the json schema, my understanding is that if you specify both values, the configuration will be rejected. Are you sure this sentence is correct?